### PR TITLE
Feature/bc 480 onboarding intro and farewell screens 

### PR DESCRIPTION
**Pull request by yeezick**

### CODE REVIEW

**Comment by yeezick** [comment]
- [ ] extending background to push footer below view height `calc(100vh - 53px)`
- [ ] match text, spacing, background with figma 
- [ ] center screen content using flexbox 
- [ ] remove onboarding text 
- [ ] resolve typeerror

File: src/layout/Layout.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 .main-content-container {
   padding-top: 93px;
-  background-color: #e5e5e5;
+  // background-color: #e5e5e5;
```
Comment (yeezick) [inline src/layout/Layout.scss:9]: remove

File: src/screens/Onboarding/Onboarding.scss [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+  margin-top: 50px;
+}
+
+.welcome-container {
+  align-items: flex-start;
+  display: inline-flex;
+  flex-direction: column;
+  gap: 5px;
+  justify-content: center;
+  padding: 0px;
```
Comment (yeezick) [inline src/screens/Onboarding/Onboarding.scss:23]: nice alphabetization :)

File: src/utils/redux/slices/userSlice.tsx [hunk 3]
```diff
@@ -30,6 +30,7 @@ const initialState: UiSliceInterface = {
       },
       profilePicture: '',
       project: '', // should be projetId
+      development
```
Comment (yeezick) [inline src/utils/redux/slices/userSlice.tsx:33]: is this intentional?

**Review by yeezick — COMMENTED** [review]
looks good beside the last 2 comments!

File: src/utils/redux/slices/userSlice.tsx (hunk 3 above)
Comment (laurenhatchett) [reply src/utils/redux/slices/userSlice.tsx:33]: this does wasn't on my local repo, I'm not sure how it got here.

**Review by laurenhatchett — COMMENTED** [review]
fixed the code based on the comments

File: src/utils/redux/slices/userSlice.tsx (hunk 3 above)
Comment (yeezick) [reply src/utils/redux/slices/userSlice.tsx:33]: could you try running git pull and see if it loads? don't wanna accidentally merge this

**Review by gooliajulia — APPROVED** [review]

**Pull request merged**